process/slash/notifier/slashingNotifier.go

```diff
@@ -0,0 +1,190 @@
+package notifier
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	coreSlash "github.com/ElrondNetwork/elrond-go-core/data/slash"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/slash"
+	"github.com/ElrondNetwork/elrond-go/state"
+	"github.com/ElrondNetwork/elrond-go/update"
+)
+
+//TODO: Move this constants to config file, when slashing notifier is integrated.
+// Please note that these are just some dummy values and are meant to be changed.
+
+// BuiltInFunctionSlashCommitmentProof = key for slashing commitment proof built-in function
+const BuiltInFunctionSlashCommitmentProof = "SlashCommitment"
+
+// CommitmentProofValue = value to issue a commitment tx proof
+const CommitmentProofValue = 1
+
+// CommitmentProofGasPrice = gas price to issue a commitment tx proof
```

Comment:
todo: move the committment gas data to config files

Reply:
There is a TODO on this file to move all constans on L21-22